setup.py

```diff
@@ -84,6 +84,8 @@
         "tornado",
         "traitlets",
         "typing;python_version<'3.7'",
+        "conda",
+        "cytoolz"
```

Comment:
I don't see where `cytoolz` is used. And as `conda` is not updated on pypi.org any more. I would prefer avoiding a direct dependency on `conda` as user may install this package with pip in non-root environments.

```suggestion
```